.github/workflows/pr-main_l1.yaml

```diff
@@ -124,6 +126,7 @@ jobs:
     needs: [docker_build]
     if: ${{ github.event_name != 'merge_group' }}
     strategy:
+      fail-fast: true
```

Comment:
this is the default, just making it explicit. Sometimes I want to change it to false when testing stuff